xml/smtp_banners.xml

```diff
@@ -1256,8 +1256,8 @@ The system or service fingerprint with the highest certainty overwrites the othe
       <param pos="0" name="system.time.format" value="EEE, dd MMM yyyy HH:mm:ss zzz"/>
       <param pos="1" name="host.name"/>
       <param pos="2" name="service.version"/>
-      <param pos="3" name="sendmail.config.version"/>
-      <param pos="4" name="sendmail.vendor.version"/>
+      <param pos="3" name="config.version"/>
```

Comment:
We should probably move all of these to `service`, so for this it would be `service.config.version`.  Leaving them without the `service` prefix makes it a bit ambiguous as to what this particular extraction is about.